pyiron/dft/job/generic.py

```diff
@@ -75,15 +75,22 @@ def exchange_correlation_functional(self, val):
             "The exchange property is not implemented for this code."
         )
 
-    def get_k_mesh_by_cell(self, cell=None, kpoints_per_angstrom=1):
+    def get_k_mesh_by_cell(self, kpoints_per_reciprocal_angstrom, cell=None):
```

Comment:
Should this function be accessed by the user or should we use a private function instead?